src/pytorch_lightning/callbacks/progress/base.py

```diff
@@ -172,6 +172,27 @@ def total_val_batches(self) -> Union[int, float]:
         assert self._trainer is not None
         return sum(self.trainer.num_val_batches) if self._trainer.fit_loop.epoch_loop._should_check_val_epoch() else 0
 
+    @property
+    def total_main_progress_bar_count_current_epoch(self) -> Union[int, float]:
```

Comment:
The fact that the main progresss bar uses it does not mean it should be named after it

```suggestion
    def total_batches_current_epoch(self) -> Union[int, float]:
```